src/sentry/sentry_apps/tasks/sentry_apps.py

```diff
@@ -326,12 +346,13 @@ def _process_resource_change(
                 )
                 if event in installation.sentry_app.events
             ]
-            data = {}
+            data: dict[str, Any] = {}
```

Comment:
Doesn't need to be addressed here, but is the  `_webhook_event_data` function typed and could we narrow this from `Any` somehow?